datastore/api/snippets.py

```diff
@@ -0,0 +1,820 @@
+# Copyright 2016, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+from collections import defaultdict
+import datetime
+from pprint import pprint
+import time
+
+import gcloud
+from gcloud import datastore
+
+
+def incomplete_key(client):
+    # [START incomplete_key]
+    key = client.key('Task')
+    # [END incomplete_key]
+
+    return key
+
+
+def named_key(client):
+    # [START named_key]
+    key = client.key('Task', 'sample_task')
+    # [END named_key]
+
+    return key
+
+
+def key_with_parent(client):
+    # [START key_with_parent]
+    key = client.key('TaskList', 'default', 'Task', 'sample_task')
+    # [END key_with_parent]
+
+    return key
+
+
+def key_with_multilevel_parent(client):
+    # [START key_with_multilevel_parent]
+    key = client.key(
+        'User', 'alice',
+        'TaskList', 'default',
+        'Task', 'sample_task')
+    # [END key_with_multilevel_parent]
+
+    return key
+
+
+def basic_entity(client):
+    # [START basic_entity]
+    task = datastore.Entity()
```

Comment:
The node.js sample didn't have one either. Though I agree, it should have a key.